### PR TITLE
Add click-loglevel

### PR DESCRIPTION
**Pull request by jwodder**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/click-loglevel/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```
Comment (dopplershift) [inline recipes/click-loglevel/LICENSE.txt:1]: `LICENSE` is already included in the PyPI tarball, so you shouldn't include it in the recipe.

File: recipes/click-loglevel/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  imports:
+    - click_loglevel
+  commands:
+    - python -m pytest -vv test
+
+about:
+  home: https://github.com/jwodder/click-loglevel
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```
Comment (dopplershift) [inline recipes/click-loglevel/meta.yaml:42]: Update to `LICENSE` which is in the PyPI tarball.

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just a couple updates about the license, otherwise this is good to go.

File: recipes/click-loglevel/meta.yaml (hunk 2 above)
Comment (jwodder) [reply recipes/click-loglevel/meta.yaml:42]: Did you mean to suggest an edit here?  Exactly what should `license_file` be changed to?

Comment (dopplershift) [reply recipes/click-loglevel/meta.yaml:42]:
```suggestion
  license_file: LICENSE
```

File: recipes/click-loglevel/LICENSE.txt (hunk 1 above)
Comment (jwodder) [reply recipes/click-loglevel/LICENSE.txt]: Removed.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**